docker/Dockerfile

```diff
@@ -30,31 +30,28 @@ RUN apt-get update && apt-get install -y \
     libxslt1-dev \
     rustc \
     zlib1g-dev \
- && rm -rf /var/lib/apt/lists/*
+    && rm -rf /var/lib/apt/lists/*
 
-# Build dependencies that are not available as wheels, to speed up rebuilds
+# Copy just what we need to pip install
+COPY scripts /synapse/scripts/
+COPY MANIFEST.in README.rst setup.py synctl /synapse/
+COPY synapse/__init__.py /synapse/synapse/__init__.py
+COPY synapse/python_dependencies.py /synapse/synapse/python_dependencies.py
```

Comment:
Could we just add a newline between this section and the comments for the next for readability?